src/components/common/BannerSizeMock/BannerSizeMock.tsx

```diff
@@ -1,36 +1,33 @@
 import { Flex, createStyles, Text } from '@mantine/core'
-import { BannerVariantNew } from 'types'
 
-const useStyles = createStyles(
-  (theme, { variant, active }: { variant: BannerVariantNew; active: boolean }) => ({
-    wrapper: {
-      display: 'flex',
-      justifyContent: 'center',
-      alignItems: 'center',
-      width: 72,
-      height: 62,
-      background: theme.colors.lightBackground[theme.fn.primaryShade()],
-      border: '1px solid',
-      borderColor: active ? 'blue' : theme.colors.decorativeBorders[theme.fn.primaryShade()],
-      borderRadius: theme.radius.sm
-    },
-    inner: {
-      width: variant.width,
-      height: variant.height,
-      backgroundColor: theme.colors.decorativeBorders[theme.fn.primaryShade()],
-      borderRadius: theme.radius.xs
-    }
-  })
-)
+const useStyles = createStyles((theme, { active }: { active: boolean }) => ({
+  wrapper: {
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    width: 72,
+    height: 62,
+    background: theme.colors.lightBackground[theme.fn.primaryShade()],
+    border: '1px solid',
+    borderColor: active ? 'blue' : theme.colors.decorativeBorders[theme.fn.primaryShade()],
+    borderRadius: theme.radius.sm
+  }
+}))
 
-const BannerSizeMock = ({ variant }: { variant: BannerVariantNew }) => {
-  const { classes } = useStyles({ variant, active: !!variant.checked })
+const BannerSizeMock = ({
+  variant,
+  active,
+  addedBannerCount
+}: {
+  variant: string
+  active: boolean
+  addedBannerCount: number | undefined
+}) => {
+  const { classes } = useStyles({ active })
   return (
     <Flex direction="column" align="center">
-      <div className={classes.wrapper}>
-        <div className={classes.inner} />
-      </div>
-      <Text size="sm">{`${variant.bannerSizes.w} x ${variant.bannerSizes.h}`}</Text>
+      <div className={classes.wrapper}>{!!addedBannerCount && <Text>{addedBannerCount}</Text>}</div>
```

Comment:
I can not run the app atm to see how it looks, but isn't it better to use the indicator from mantine - https://v6.mantine.dev/core/indicator/